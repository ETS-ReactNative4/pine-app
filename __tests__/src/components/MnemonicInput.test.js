import React from 'react';
import renderer from 'react-test-renderer';
import MnemonicInput from '../../../src/components/MnemonicInput';

describe('MnemonicInput', () => {
  it('renders correctly', () => {
    const correctPhrase = 'during bulb nominee acquire paddle next course stable govern eagle title wing';

    const tree = renderer.create(
      <MnemonicInput correctPhrase={correctPhrase} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
