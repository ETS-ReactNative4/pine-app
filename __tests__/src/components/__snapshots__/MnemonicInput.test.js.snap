// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MnemonicInput renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "flexWrap": "wrap",
        "justifyContent": "space-between",
      },
      undefined,
    ]
  }
>
  <View
    style={
      Object {
        "padding": 4,
        "width": "50%",
      }
    }
  >
    <View>
      <View
        style={
          Array [
            Object {
              "backgroundColor": "white",
              "borderRadius": 27,
              "borderWidth": 1,
              "padding": 17,
              "paddingBottom": 3,
              "paddingTop": 7,
              "shadowColor": "#000000",
              "shadowOffset": Object {
                "height": 6,
                "width": 0,
              },
              "shadowOpacity": 0.02,
              "shadowRadius": 5,
            },
            Object {
              "paddingTop": 6,
            },
            Object {
              "borderColor": "#DADADA",
            },
          ]
        }
      >
        <TextInput
          allowFontScaling={true}
          autoCapitalize="none"
          autoCorrect={false}
          enablesReturnKeyAutomatically={true}
          keyboardType={undefined}
          onBlur={[Function]}
          onChangeText={[Function]}
          onFocus={[Function]}
          onSubmitEditing={[Function]}
          placeholder={undefined}
          placeholderTextColor="#DADADA"
          returnKeyType="next"
          selectionColor="#3FA7D6"
          style={
            Array [
              Object {
                "color": "#000000",
                "fontSize": 13,
                "height": 27,
                "letterSpacing": 1.45,
                "width": 278,
              },
              Array [
                Object {
                  "fontFamily": "Menlo-Regular",
                  "fontSize": 13,
                  "height": 27,
                  "padding": 0,
                  "paddingLeft": 16,
                  "width": "100%",
                },
                Object {},
              ],
            ]
          }
          value={undefined}
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        numberOfLines={undefined}
        style={
          Array [
            Object {
              "color": "#000000",
              "fontFamily": "System",
              "fontWeight": "400",
            },
            Array [
              Object {
                "color": "#DDDDDF",
                "fontSize": 13,
                "fontWeight": "bold",
                "left": 7,
                "position": "absolute",
                "textAlign": "right",
                "top": 11.5,
                "width": 20,
              },
              Object {},
            ],
          ]
        }
      >
        1
        .
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "padding": 4,
        "width": "50%",
      }
    }
  >
    <View>
      <View
        style={
          Array [
            Object {
              "backgroundColor": "white",
              "borderRadius": 27,
              "borderWidth": 1,
              "padding": 17,
              "paddingBottom": 3,
              "paddingTop": 7,
              "shadowColor": "#000000",
              "shadowOffset": Object {
                "height": 6,
                "width": 0,
              },
              "shadowOpacity": 0.02,
              "shadowRadius": 5,
            },
            Object {
              "paddingTop": 6,
            },
            Object {
              "borderColor": "#DADADA",
            },
          ]
        }
      >
        <TextInput
          allowFontScaling={true}
          autoCapitalize="none"
          autoCorrect={false}
          enablesReturnKeyAutomatically={true}
          keyboardType={undefined}
          onBlur={[Function]}
          onChangeText={[Function]}
          onFocus={[Function]}
          onSubmitEditing={[Function]}
          placeholder={undefined}
          placeholderTextColor="#DADADA"
          returnKeyType="next"
          selectionColor="#3FA7D6"
          style={
            Array [
              Object {
                "color": "#000000",
                "fontSize": 13,
                "height": 27,
                "letterSpacing": 1.45,
                "width": 278,
              },
              Array [
                Object {
                  "fontFamily": "Menlo-Regular",
                  "fontSize": 13,
                  "height": 27,
                  "padding": 0,
                  "paddingLeft": 16,
                  "width": "100%",
                },
                Object {},
              ],
            ]
          }
          value={undefined}
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        numberOfLines={undefined}
        style={
          Array [
            Object {
              "color": "#000000",
              "fontFamily": "System",
              "fontWeight": "400",
            },
            Array [
              Object {
                "color": "#DDDDDF",
                "fontSize": 13,
                "fontWeight": "bold",
                "left": 7,
                "position": "absolute",
                "textAlign": "right",
                "top": 11.5,
                "width": 20,
              },
              Object {},
            ],
          ]
        }
      >
        2
        .
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "padding": 4,
        "width": "50%",
      }
    }
  >
    <View>
      <View
        style={
          Array [
            Object {
              "backgroundColor": "white",
              "borderRadius": 27,
              "borderWidth": 1,
              "padding": 17,
              "paddingBottom": 3,
              "paddingTop": 7,
              "shadowColor": "#000000",
              "shadowOffset": Object {
                "height": 6,
                "width": 0,
              },
              "shadowOpacity": 0.02,
              "shadowRadius": 5,
            },
            Object {
              "paddingTop": 6,
            },
            Object {
              "borderColor": "#DADADA",
            },
          ]
        }
      >
        <TextInput
          allowFontScaling={true}
          autoCapitalize="none"
          autoCorrect={false}
          enablesReturnKeyAutomatically={true}
          keyboardType={undefined}
          onBlur={[Function]}
          onChangeText={[Function]}
          onFocus={[Function]}
          onSubmitEditing={[Function]}
          placeholder={undefined}
          placeholderTextColor="#DADADA"
          returnKeyType="next"
          selectionColor="#3FA7D6"
          style={
            Array [
              Object {
                "color": "#000000",
                "fontSize": 13,
                "height": 27,
                "letterSpacing": 1.45,
                "width": 278,
              },
              Array [
                Object {
                  "fontFamily": "Menlo-Regular",
                  "fontSize": 13,
                  "height": 27,
                  "padding": 0,
                  "paddingLeft": 16,
                  "width": "100%",
                },
                Object {},
              ],
            ]
          }
          value={undefined}
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        numberOfLines={undefined}
        style={
          Array [
            Object {
              "color": "#000000",
              "fontFamily": "System",
              "fontWeight": "400",
            },
            Array [
              Object {
                "color": "#DDDDDF",
                "fontSize": 13,
                "fontWeight": "bold",
                "left": 7,
                "position": "absolute",
                "textAlign": "right",
                "top": 11.5,
                "width": 20,
              },
              Object {},
            ],
          ]
        }
      >
        3
        .
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "padding": 4,
        "width": "50%",
      }
    }
  >
    <View>
      <View
        style={
          Array [
            Object {
              "backgroundColor": "white",
              "borderRadius": 27,
              "borderWidth": 1,
              "padding": 17,
              "paddingBottom": 3,
              "paddingTop": 7,
              "shadowColor": "#000000",
              "shadowOffset": Object {
                "height": 6,
                "width": 0,
              },
              "shadowOpacity": 0.02,
              "shadowRadius": 5,
            },
            Object {
              "paddingTop": 6,
            },
            Object {
              "borderColor": "#DADADA",
            },
          ]
        }
      >
        <TextInput
          allowFontScaling={true}
          autoCapitalize="none"
          autoCorrect={false}
          enablesReturnKeyAutomatically={true}
          keyboardType={undefined}
          onBlur={[Function]}
          onChangeText={[Function]}
          onFocus={[Function]}
          onSubmitEditing={[Function]}
          placeholder={undefined}
          placeholderTextColor="#DADADA"
          returnKeyType="next"
          selectionColor="#3FA7D6"
          style={
            Array [
              Object {
                "color": "#000000",
                "fontSize": 13,
                "height": 27,
                "letterSpacing": 1.45,
                "width": 278,
              },
              Array [
                Object {
                  "fontFamily": "Menlo-Regular",
                  "fontSize": 13,
                  "height": 27,
                  "padding": 0,
                  "paddingLeft": 16,
                  "width": "100%",
                },
                Object {},
              ],
            ]
          }
          value={undefined}
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        numberOfLines={undefined}
        style={
          Array [
            Object {
              "color": "#000000",
              "fontFamily": "System",
              "fontWeight": "400",
            },
            Array [
              Object {
                "color": "#DDDDDF",
                "fontSize": 13,
                "fontWeight": "bold",
                "left": 7,
                "position": "absolute",
                "textAlign": "right",
                "top": 11.5,
                "width": 20,
              },
              Object {},
            ],
          ]
        }
      >
        4
        .
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "padding": 4,
        "width": "50%",
      }
    }
  >
    <View>
      <View
        style={
          Array [
            Object {
              "backgroundColor": "white",
              "borderRadius": 27,
              "borderWidth": 1,
              "padding": 17,
              "paddingBottom": 3,
              "paddingTop": 7,
              "shadowColor": "#000000",
              "shadowOffset": Object {
                "height": 6,
                "width": 0,
              },
              "shadowOpacity": 0.02,
              "shadowRadius": 5,
            },
            Object {
              "paddingTop": 6,
            },
            Object {
              "borderColor": "#DADADA",
            },
          ]
        }
      >
        <TextInput
          allowFontScaling={true}
          autoCapitalize="none"
          autoCorrect={false}
          enablesReturnKeyAutomatically={true}
          keyboardType={undefined}
          onBlur={[Function]}
          onChangeText={[Function]}
          onFocus={[Function]}
          onSubmitEditing={[Function]}
          placeholder={undefined}
          placeholderTextColor="#DADADA"
          returnKeyType="next"
          selectionColor="#3FA7D6"
          style={
            Array [
              Object {
                "color": "#000000",
                "fontSize": 13,
                "height": 27,
                "letterSpacing": 1.45,
                "width": 278,
              },
              Array [
                Object {
                  "fontFamily": "Menlo-Regular",
                  "fontSize": 13,
                  "height": 27,
                  "padding": 0,
                  "paddingLeft": 16,
                  "width": "100%",
                },
                Object {},
              ],
            ]
          }
          value={undefined}
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        numberOfLines={undefined}
        style={
          Array [
            Object {
              "color": "#000000",
              "fontFamily": "System",
              "fontWeight": "400",
            },
            Array [
              Object {
                "color": "#DDDDDF",
                "fontSize": 13,
                "fontWeight": "bold",
                "left": 7,
                "position": "absolute",
                "textAlign": "right",
                "top": 11.5,
                "width": 20,
              },
              Object {},
            ],
          ]
        }
      >
        5
        .
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "padding": 4,
        "width": "50%",
      }
    }
  >
    <View>
      <View
        style={
          Array [
            Object {
              "backgroundColor": "white",
              "borderRadius": 27,
              "borderWidth": 1,
              "padding": 17,
              "paddingBottom": 3,
              "paddingTop": 7,
              "shadowColor": "#000000",
              "shadowOffset": Object {
                "height": 6,
                "width": 0,
              },
              "shadowOpacity": 0.02,
              "shadowRadius": 5,
            },
            Object {
              "paddingTop": 6,
            },
            Object {
              "borderColor": "#DADADA",
            },
          ]
        }
      >
        <TextInput
          allowFontScaling={true}
          autoCapitalize="none"
          autoCorrect={false}
          enablesReturnKeyAutomatically={true}
          keyboardType={undefined}
          onBlur={[Function]}
          onChangeText={[Function]}
          onFocus={[Function]}
          onSubmitEditing={[Function]}
          placeholder={undefined}
          placeholderTextColor="#DADADA"
          returnKeyType="next"
          selectionColor="#3FA7D6"
          style={
            Array [
              Object {
                "color": "#000000",
                "fontSize": 13,
                "height": 27,
                "letterSpacing": 1.45,
                "width": 278,
              },
              Array [
                Object {
                  "fontFamily": "Menlo-Regular",
                  "fontSize": 13,
                  "height": 27,
                  "padding": 0,
                  "paddingLeft": 16,
                  "width": "100%",
                },
                Object {},
              ],
            ]
          }
          value={undefined}
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        numberOfLines={undefined}
        style={
          Array [
            Object {
              "color": "#000000",
              "fontFamily": "System",
              "fontWeight": "400",
            },
            Array [
              Object {
                "color": "#DDDDDF",
                "fontSize": 13,
                "fontWeight": "bold",
                "left": 7,
                "position": "absolute",
                "textAlign": "right",
                "top": 11.5,
                "width": 20,
              },
              Object {},
            ],
          ]
        }
      >
        6
        .
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "padding": 4,
        "width": "50%",
      }
    }
  >
    <View>
      <View
        style={
          Array [
            Object {
              "backgroundColor": "white",
              "borderRadius": 27,
              "borderWidth": 1,
              "padding": 17,
              "paddingBottom": 3,
              "paddingTop": 7,
              "shadowColor": "#000000",
              "shadowOffset": Object {
                "height": 6,
                "width": 0,
              },
              "shadowOpacity": 0.02,
              "shadowRadius": 5,
            },
            Object {
              "paddingTop": 6,
            },
            Object {
              "borderColor": "#DADADA",
            },
          ]
        }
      >
        <TextInput
          allowFontScaling={true}
          autoCapitalize="none"
          autoCorrect={false}
          enablesReturnKeyAutomatically={true}
          keyboardType={undefined}
          onBlur={[Function]}
          onChangeText={[Function]}
          onFocus={[Function]}
          onSubmitEditing={[Function]}
          placeholder={undefined}
          placeholderTextColor="#DADADA"
          returnKeyType="next"
          selectionColor="#3FA7D6"
          style={
            Array [
              Object {
                "color": "#000000",
                "fontSize": 13,
                "height": 27,
                "letterSpacing": 1.45,
                "width": 278,
              },
              Array [
                Object {
                  "fontFamily": "Menlo-Regular",
                  "fontSize": 13,
                  "height": 27,
                  "padding": 0,
                  "paddingLeft": 16,
                  "width": "100%",
                },
                Object {},
              ],
            ]
          }
          value={undefined}
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        numberOfLines={undefined}
        style={
          Array [
            Object {
              "color": "#000000",
              "fontFamily": "System",
              "fontWeight": "400",
            },
            Array [
              Object {
                "color": "#DDDDDF",
                "fontSize": 13,
                "fontWeight": "bold",
                "left": 7,
                "position": "absolute",
                "textAlign": "right",
                "top": 11.5,
                "width": 20,
              },
              Object {},
            ],
          ]
        }
      >
        7
        .
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "padding": 4,
        "width": "50%",
      }
    }
  >
    <View>
      <View
        style={
          Array [
            Object {
              "backgroundColor": "white",
              "borderRadius": 27,
              "borderWidth": 1,
              "padding": 17,
              "paddingBottom": 3,
              "paddingTop": 7,
              "shadowColor": "#000000",
              "shadowOffset": Object {
                "height": 6,
                "width": 0,
              },
              "shadowOpacity": 0.02,
              "shadowRadius": 5,
            },
            Object {
              "paddingTop": 6,
            },
            Object {
              "borderColor": "#DADADA",
            },
          ]
        }
      >
        <TextInput
          allowFontScaling={true}
          autoCapitalize="none"
          autoCorrect={false}
          enablesReturnKeyAutomatically={true}
          keyboardType={undefined}
          onBlur={[Function]}
          onChangeText={[Function]}
          onFocus={[Function]}
          onSubmitEditing={[Function]}
          placeholder={undefined}
          placeholderTextColor="#DADADA"
          returnKeyType="next"
          selectionColor="#3FA7D6"
          style={
            Array [
              Object {
                "color": "#000000",
                "fontSize": 13,
                "height": 27,
                "letterSpacing": 1.45,
                "width": 278,
              },
              Array [
                Object {
                  "fontFamily": "Menlo-Regular",
                  "fontSize": 13,
                  "height": 27,
                  "padding": 0,
                  "paddingLeft": 16,
                  "width": "100%",
                },
                Object {},
              ],
            ]
          }
          value={undefined}
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        numberOfLines={undefined}
        style={
          Array [
            Object {
              "color": "#000000",
              "fontFamily": "System",
              "fontWeight": "400",
            },
            Array [
              Object {
                "color": "#DDDDDF",
                "fontSize": 13,
                "fontWeight": "bold",
                "left": 7,
                "position": "absolute",
                "textAlign": "right",
                "top": 11.5,
                "width": 20,
              },
              Object {},
            ],
          ]
        }
      >
        8
        .
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "padding": 4,
        "width": "50%",
      }
    }
  >
    <View>
      <View
        style={
          Array [
            Object {
              "backgroundColor": "white",
              "borderRadius": 27,
              "borderWidth": 1,
              "padding": 17,
              "paddingBottom": 3,
              "paddingTop": 7,
              "shadowColor": "#000000",
              "shadowOffset": Object {
                "height": 6,
                "width": 0,
              },
              "shadowOpacity": 0.02,
              "shadowRadius": 5,
            },
            Object {
              "paddingTop": 6,
            },
            Object {
              "borderColor": "#DADADA",
            },
          ]
        }
      >
        <TextInput
          allowFontScaling={true}
          autoCapitalize="none"
          autoCorrect={false}
          enablesReturnKeyAutomatically={true}
          keyboardType={undefined}
          onBlur={[Function]}
          onChangeText={[Function]}
          onFocus={[Function]}
          onSubmitEditing={[Function]}
          placeholder={undefined}
          placeholderTextColor="#DADADA"
          returnKeyType="next"
          selectionColor="#3FA7D6"
          style={
            Array [
              Object {
                "color": "#000000",
                "fontSize": 13,
                "height": 27,
                "letterSpacing": 1.45,
                "width": 278,
              },
              Array [
                Object {
                  "fontFamily": "Menlo-Regular",
                  "fontSize": 13,
                  "height": 27,
                  "padding": 0,
                  "paddingLeft": 16,
                  "width": "100%",
                },
                Object {},
              ],
            ]
          }
          value={undefined}
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        numberOfLines={undefined}
        style={
          Array [
            Object {
              "color": "#000000",
              "fontFamily": "System",
              "fontWeight": "400",
            },
            Array [
              Object {
                "color": "#DDDDDF",
                "fontSize": 13,
                "fontWeight": "bold",
                "left": 7,
                "position": "absolute",
                "textAlign": "right",
                "top": 11.5,
                "width": 20,
              },
              Object {},
            ],
          ]
        }
      >
        9
        .
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "padding": 4,
        "width": "50%",
      }
    }
  >
    <View>
      <View
        style={
          Array [
            Object {
              "backgroundColor": "white",
              "borderRadius": 27,
              "borderWidth": 1,
              "padding": 17,
              "paddingBottom": 3,
              "paddingTop": 7,
              "shadowColor": "#000000",
              "shadowOffset": Object {
                "height": 6,
                "width": 0,
              },
              "shadowOpacity": 0.02,
              "shadowRadius": 5,
            },
            Object {
              "paddingTop": 6,
            },
            Object {
              "borderColor": "#DADADA",
            },
          ]
        }
      >
        <TextInput
          allowFontScaling={true}
          autoCapitalize="none"
          autoCorrect={false}
          enablesReturnKeyAutomatically={true}
          keyboardType={undefined}
          onBlur={[Function]}
          onChangeText={[Function]}
          onFocus={[Function]}
          onSubmitEditing={[Function]}
          placeholder={undefined}
          placeholderTextColor="#DADADA"
          returnKeyType="next"
          selectionColor="#3FA7D6"
          style={
            Array [
              Object {
                "color": "#000000",
                "fontSize": 13,
                "height": 27,
                "letterSpacing": 1.45,
                "width": 278,
              },
              Array [
                Object {
                  "fontFamily": "Menlo-Regular",
                  "fontSize": 13,
                  "height": 27,
                  "padding": 0,
                  "paddingLeft": 16,
                  "width": "100%",
                },
                Object {},
              ],
            ]
          }
          value={undefined}
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        numberOfLines={undefined}
        style={
          Array [
            Object {
              "color": "#000000",
              "fontFamily": "System",
              "fontWeight": "400",
            },
            Array [
              Object {
                "color": "#DDDDDF",
                "fontSize": 13,
                "fontWeight": "bold",
                "left": 7,
                "position": "absolute",
                "textAlign": "right",
                "top": 11.5,
                "width": 20,
              },
              Object {},
            ],
          ]
        }
      >
        10
        .
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "padding": 4,
        "width": "50%",
      }
    }
  >
    <View>
      <View
        style={
          Array [
            Object {
              "backgroundColor": "white",
              "borderRadius": 27,
              "borderWidth": 1,
              "padding": 17,
              "paddingBottom": 3,
              "paddingTop": 7,
              "shadowColor": "#000000",
              "shadowOffset": Object {
                "height": 6,
                "width": 0,
              },
              "shadowOpacity": 0.02,
              "shadowRadius": 5,
            },
            Object {
              "paddingTop": 6,
            },
            Object {
              "borderColor": "#DADADA",
            },
          ]
        }
      >
        <TextInput
          allowFontScaling={true}
          autoCapitalize="none"
          autoCorrect={false}
          enablesReturnKeyAutomatically={true}
          keyboardType={undefined}
          onBlur={[Function]}
          onChangeText={[Function]}
          onFocus={[Function]}
          onSubmitEditing={[Function]}
          placeholder={undefined}
          placeholderTextColor="#DADADA"
          returnKeyType="next"
          selectionColor="#3FA7D6"
          style={
            Array [
              Object {
                "color": "#000000",
                "fontSize": 13,
                "height": 27,
                "letterSpacing": 1.45,
                "width": 278,
              },
              Array [
                Object {
                  "fontFamily": "Menlo-Regular",
                  "fontSize": 13,
                  "height": 27,
                  "padding": 0,
                  "paddingLeft": 16,
                  "width": "100%",
                },
                Object {},
              ],
            ]
          }
          value={undefined}
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        numberOfLines={undefined}
        style={
          Array [
            Object {
              "color": "#000000",
              "fontFamily": "System",
              "fontWeight": "400",
            },
            Array [
              Object {
                "color": "#DDDDDF",
                "fontSize": 13,
                "fontWeight": "bold",
                "left": 7,
                "position": "absolute",
                "textAlign": "right",
                "top": 11.5,
                "width": 20,
              },
              Object {},
            ],
          ]
        }
      >
        11
        .
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "padding": 4,
        "width": "50%",
      }
    }
  >
    <View>
      <View
        style={
          Array [
            Object {
              "backgroundColor": "white",
              "borderRadius": 27,
              "borderWidth": 1,
              "padding": 17,
              "paddingBottom": 3,
              "paddingTop": 7,
              "shadowColor": "#000000",
              "shadowOffset": Object {
                "height": 6,
                "width": 0,
              },
              "shadowOpacity": 0.02,
              "shadowRadius": 5,
            },
            Object {
              "paddingTop": 6,
            },
            Object {
              "borderColor": "#DADADA",
            },
          ]
        }
      >
        <TextInput
          allowFontScaling={true}
          autoCapitalize="none"
          autoCorrect={false}
          enablesReturnKeyAutomatically={true}
          keyboardType={undefined}
          onBlur={[Function]}
          onChangeText={[Function]}
          onFocus={[Function]}
          onSubmitEditing={[Function]}
          placeholder={undefined}
          placeholderTextColor="#DADADA"
          returnKeyType="done"
          selectionColor="#3FA7D6"
          style={
            Array [
              Object {
                "color": "#000000",
                "fontSize": 13,
                "height": 27,
                "letterSpacing": 1.45,
                "width": 278,
              },
              Array [
                Object {
                  "fontFamily": "Menlo-Regular",
                  "fontSize": 13,
                  "height": 27,
                  "padding": 0,
                  "paddingLeft": 16,
                  "width": "100%",
                },
                Object {},
              ],
            ]
          }
          value={undefined}
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        numberOfLines={undefined}
        style={
          Array [
            Object {
              "color": "#000000",
              "fontFamily": "System",
              "fontWeight": "400",
            },
            Array [
              Object {
                "color": "#DDDDDF",
                "fontSize": 13,
                "fontWeight": "bold",
                "left": 7,
                "position": "absolute",
                "textAlign": "right",
                "top": 11.5,
                "width": 20,
              },
              Object {},
            ],
          ]
        }
      >
        12
        .
      </Text>
    </View>
  </View>
</View>
`;
